src/brand/bhyve/support

```diff
@@ -50,7 +50,7 @@ case $cmd in
 		;;
 	    $ZONE_STATE_RUNNING:$ZONE_STATE_CMD_HALT)
 		# halting (even as part of rebooting)
-		bhyve_shutdown 0
+		bhyve_shutdown 1800
```

Comment:
half an hour ? seems a bit excessive

Reply:
Before this we had no limit, and Windows guests can take a long time to shut down if they are applying updates.
I think 30 minutes is probably reasonable.